examples/src/main/scala/org/apache/spark/examples/ml/ChiSqSelectorExample.scala

```diff
@@ -30,9 +30,10 @@ object ChiSqSelectorExample {
       .builder
       .appName("ChiSqSelectorExample")
       .getOrCreate()
-    import spark.implicits._
 
     // $example on$
+    import spark.implicits._
```

Comment:
I think this isn't required when you run a Spark shell. Looks intentionally excluded given that `val spark` initialization isn't involved already.

Reply:
Ok, I'll go ahead and close the PR.  Thanks for the review.